### ISSUE TITLE
ModuleNotFoundError: No module named 'markdown.extensions.pymdownx'

### ISSUE THREAD

**Issue by MartinThoma** [ncornette/Python-Markdown-Editor#21]
Directly after installing it

**Comment by jnezbeda** [comment]
Same problem:
```
pip3 install markdown-editor

Requirement already satisfied: markdown-editor in /usr/local/lib/python3.8/dist-packages (1.0.4)
Requirement already satisfied: bottle in /usr/local/lib/python3.8/dist-packages (from markdown-editor) (0.12.19)
Requirement already satisfied: pymdown-extensions>=2.0 in /usr/local/lib/python3.8/dist-packages (from markdown-editor) (8.0.1)
Requirement already satisfied: Markdown in /usr/lib/python3/dist-packages (from markdown-editor) (3.1.1)
Requirement already satisfied: Pygments in /usr/lib/python3/dist-packages (from markdown-editor) (2.3.1)

markdown_edit

Traceback (most recent call last):
  File "/usr/local/bin/markdown_edit", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.8/dist-packages/markdown_edit.py", line 95, in main
    markdown_processor = markdown.Markdown(**options)
  File "/usr/lib/python3/dist-packages/markdown/core.py", line 99, in __init__
    self.registerExtensions(extensions=kwargs.get('extensions', []),
  File "/usr/lib/python3/dist-packages/markdown/core.py", line 126, in registerExtensions
    ext = self.build_extension(ext, configs.get(ext, {}))
  File "/usr/lib/python3/dist-packages/markdown/core.py", line 166, in build_extension
    module = importlib.import_module(ext_name)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 973, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'pymdownx.extrarawhtml'

```

**Comment by h-marti** [comment]
Same for me

**Comment by jlinoff** [comment]
Not sure if this helps but i was able to fix this by running

```bash
pip install 'pymdown-extensions<8.0,>7.0'
```

**Comment by ncornette** [comment]
Fixed in version 1.0.6 (thank you for comment)
